code/components/extra-natives-five/src/VehicleExtraNatives.cpp

```diff
@@ -316,6 +316,7 @@ static int VehicleDamageStructOffset;
 static bool* g_trainsForceDoorsOpen;
 static int TrainDoorCountOffset;
 static int TrainDoorArrayPointerOffset;
+static int g_trainFlagOffset;
```

Comment:
Nit: while the global style guide is `g_trainFlagOffset` - keeping consistent style inside a single file is a priority. Lets use `TrainFlagOffset` in this case :).

Reply:
This file is a bit of a mess in terms of style. But I've changed it to TrainFlagOffset